README.md

```diff
@@ -1,8 +1,8 @@
 # VC API Test Suite Implementations _(vc-api-test-suite-implementations)_
 
-[![Build status](https://img.shields.io/github/workflow/status/digitalbazaar/vc-api-test-suite-implementations/Node.js%20CI)](https://github.com/digitalbazaar/vc-api-test-suite-implementations/actions?query=workflow%3A%22Node.js+CI%22)
-[![Coverage status](https://img.shields.io/codecov/c/github/digitalbazaar/vc-api-test-suite-implementations)](https://codecov.io/gh/digitalbazaar/vc-api-test-suite-implementations)
-[![NPM Version](https://img.shields.io/npm/v/@digitalbazaar/vc-api-test-suite-implementations.svg)](https://npm.im/@digitalbazaar/vc-api-test-suite-implementations)
+[![Build status](https://img.shields.io/github/workflow/status/w3c-ccg/vc-api-test-suite-implementations/Node.js%20CI)](https://github.com/digitalbazaar/vc-api-test-suite-implementations/actions?query=workflow%3A%22Node.js+CI%22)
+[![Coverage status](https://img.shields.io/codecov/c/github/w3c-ccg/vc-api-test-suite-implementations)](https://codecov.io/gh/digitalbazaar/vc-api-test-suite-implementations)
```

Comment:
There are still `digitalbazaar` keywords in the urls `https://github.com/digitalbazaar/vc-api-test-suite-implementations/actions?query=workflow%3A%22Node.js+CI%22` and `https://codecov.io/gh/digitalbazaar/vc-api-test-suite-implementations`.